data/7.3/Dockerfile

```diff
@@ -20,17 +20,8 @@ RUN pickle install redis --no-interaction && docker-php-ext-enable redis
 RUN pecl install xdebug && docker-php-ext-enable xdebug
 
 
-# install Microsoft ODBC drivers
-# based on https://github.com/microsoft/msphpsql/issues/300#issuecomment-673143369
-RUN curl -O https://download.microsoft.com/download/e/4/e/e4e67866-dffd-428c-aac7-8d28ddafb39b/msodbcsql17_17.6.1.1-1_amd64.apk -sS && \
```

Comment:
not sure if this can be removed, must be tested with real app, but let's merge it...

Reply:
ok